Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -107,6 +107,18 @@
   price: 2
   icon: /Textures/Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_sl.rsi/icon.png
 
+#Utility
+
+- type: uplinkListing
+  id: UplinkHoloparaKit
+  category: Utility
+  itemId: BoxHoloparasite
```

Comment:
So the `BoxHoloparasite` appears to just not exist? Is it meant to be a box with the injector and manual?

Also, `UplinkStoreListingPrototype` should have a custom type serializer.

Reply:
Agreed, making a separate PR.